apps/public-docsite-v9/src/Concepts/AdvancedConfiguration.stories.mdx

```diff
@@ -29,3 +31,21 @@ function MyComponent(props) {
 ```
 
 You can check complete example at [CodeSandbox](https://codesandbox.io/s/fluentuireact-components-render-into-iframe-l62ke).
+
+### Content Security Policies
+
+To add `nonce` attribute need for [Content Security Policies](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP), please use `styleElementAttributes` to specify it:
+
+```js
+import { createDOMRenderer } from '@fluentui/react-components';
+
+const renderer = createDOMRenderer(document, {
```

Comment:
I would also add the usage of the renderer with the provider in this example

Reply:
Done 👍